tests/audit-tests/run-tests.sh

```diff
@@ -5,36 +5,7 @@ set -eu
 gradle -v
 
 echo "Current Environment $TEST_ENVIRONMENT"
-#./gradlew clean test -Dcucumber.filter.tags="@$TEST_ENVIRONMENT"
 
-cat <<EOF > "$TEST_REPORT_DIR/result.json"
-[
-  {
-    "uri": "test.sh",
-    "name": "Acceptance test",
-    "elements": [
-      {
-        "type": "scenario",
-        "name": "API Gateway request",
-        "line": 6,
-        "steps": [
-          {
-            "keyword": "Given ",
-            "name": "this step fails",
-            "line": 6,
-            "match": {
-              "location": "test.sh:4"
-            },
-            "result": {
-              "status": "passed",
-              "duration": 1
-            }
-          }
-        ]
-      }
-    ]
-  }
-]
-EOF
+./gradlew clean test -PTEST_REPORT_DIR=$TEST_REPORT_DIR
```

Comment:
```suggestion
./gradlew clean test
```
As the other comment, we can pass the variable directly in